src/graphics/EInkDisplay2.cpp

```diff
@@ -55,7 +55,7 @@ GxEPD2_BW<TECHO_DISPLAY_MODEL, TECHO_DISPLAY_MODEL::HEIGHT> *adafruitDisplay;
 EInkDisplay::EInkDisplay(uint8_t address, int sda, int scl, OLEDDISPLAY_GEOMETRY geometry, HW_I2C i2cBus)
 {
 #if defined(TTGO_T_ECHO)
-    setGeometry(GEOMETRY_RAWMODE, TECHO_DISPLAY_MODEL::WIDTH, TECHO_DISPLAY_MODEL::HEIGHT);
```

Comment:
Is this change necessary?

Reply:
I was looking at all the e-ink support as raised by the previous ESP32-S3-PICO PR. I wanted to look at supply these geometry values in the platformio.ini variant files which would reduce the checks so it ok for now

Reply:
This was the change actually that made it all work for me (aside from the explicitly partial update parts). The why is an open question to me though